clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -323,7 +323,7 @@ public KafkaStoreIngestionService(
         .setPubSubTopicRepository(pubSubTopicRepository)
         .setMetricsRepository(metricsRepository)
         .setLocalKafkaBootstrapServers(serverConfig.getKafkaBootstrapServers())
-        .setPubSubConsumerAdapterFactory(new ApacheKafkaConsumerAdapterFactory())
```

Comment:
Can you replace all instances of `Apache*AdapterFactory` with the corresponding `pubSubClientsFactory.get*AdapterFactory()`? For example, L423.

Reply:
I realized we need to add config for all client adapters setting, so that `DavinciBackend` and `IngestionIsolationServer` could pick up those from config.